tests/test_app.py

```diff
@@ -288,6 +288,64 @@ def test_get_title_returns_page_containing_official_copy_info_page_when_present(
                 assert response_body.index(string_to_find) > response_body.index(previous_string)
 
 
+class TestDisplayTitlePdf(BaseServerTest):
+
+    def setup_method(self, method):
+        self.app = app.test_client()
+        self._log_in_user()
+
+        with mock.patch('service.login_api_client.authenticate_user', return_value=True):
+            self._log_in_user()
+
+    @mock.patch('service.api_client.requests.get', return_value=unavailable_title)
+    def test_display_title_pdf_no_title(self, mock_get):
+        response = self.app.get('/titles/titleref.pdf')
+        assert response.status_code == 404
+        assert 'Page not found' in response.data.decode()
+
+    @mock.patch('service.api_client.requests.get', return_value=fake_title)
+    @mock.patch('service.api_client.get_official_copy_data', return_value=official_copy_response)
+    def test_display_title_pdf(self, mock_get_official_copy_data, mock_get):
+        response = self.app.get('/titles/titleref.pdf')
+        assert response.status_code == 200
```

Comment:
These tests are good. It would be worth checking if we return the content returned from the renderer. Currently we only check if we get a 200 response.